examples/README.md

```diff
@@ -0,0 +1,113 @@
+# Examples
+
+this directory contains examples for a few features of the autograder
+
+## Assessments
+
+The Hello folder and Hello.tar contain various examples of different styles of Assessments
+
+
+## Importing Assessments
+
+whether you are importing one or more assessments the procedure is the same
+
+multiple assessment import starter.tar is the starter file for importing one or more assessments
+
+
+```bash
+multiple assessment import starter.tar
+├── assessment1
+│   ├── assessment1.yml
```

Comment:
Should this be `assessmen1.yml` or `properties.yml`?

Reply:
yep updated